book/website/community-handbook/translation/translation-hello-crowdin.md

```diff
@@ -0,0 +1,165 @@
+(ch-translation-hello-crowdin)=
+
+## Crowdin 101
+
+Crowdin editor is your friend.
+You can use it to change translation language, proofread, add comments for contributors, contact the managers, vote translations, view suggestions for translation from translation memory or find Machine Translation from Google, Crowdin, DeepL, and others.
+
+### Hello Crowdin
+
+<<<<<<< HEAD
+Once you click on any file, you will be directed to the comfortable mode in Crowdin crowdsourcing editor. 
+There are different modes and editors inside Crowdin but we will only go through comfortable mode and proofreader mode in Crowdin crowdsourcing editor.
+You can find more information about Crowdin Editor from the [documentation here](https://support.crowdin.com/enterprise/getting-started-for-translators/).
+
+The comfortable model is divided into four sections:
+=======
+Once you click on any file, you will be directed to the comfortable mode in Crowdin crowdsourcing editor.
+There are different modes and editors inside Crowdin but we will only go through comfortable mode and proofreader mode in Crowdin crowdsourcing editor.
+You can find more information about Crowdin Editor from the [documentation here](https://support.crowdin.com/enterprise/getting-started-for-translators/).
+
+The comfortable mode is divided into four sections:
+>>>>>>> 85d5e6153aefe29281d08b04d5a255e210dd8808
+1. **Left Sidebar:** It contains all strings in the file that you will translate.
+2. **Middle-top area:** The main working area where you edit/upvote the translations.
+3. **Middle-bottom area:** This section contains suggestions from Translation Memory, Machine Translation (MT) suggestions, and translations by other project participants
+4. **Right sidebar:** You can use it to add comments, report issues, and see the existing Glossary available for the strings.
+
+
+```{figure} ../../figures/crowdin-editor.png
+---
+name: crowdin-editor
+width: 90%
+alt: The crowdin editor with four sections labelled from number 1-4.
+---
+```  
+
+<<<<<<< HEAD
+As shown in the image below, the Middle-top area (3) is the main working area with the source string on the top, and the section where you can type in translations below. 
+=======
+As shown in the image below, the Middle-top area (3) is the main working area with the source string on the top, and the section where you can type in translations below.
+>>>>>>> 85d5e6153aefe29281d08b04d5a255e210dd8808
+Crowdin will show you suggestions for translation carried out using three different engines (Google Translate, Crowdin Translate, DeepL), which will show you several possible translations that you can further edit.
+
+```{admonition} Add Translation Engine
+:class: tip
+If your language of interest works better with different engines (e.g. Microsoft Translator, Yandex.Translate, Amazon Translate), please request it and we will do our best to integrate it.  
```

Comment:
please replace e.g with for example, or the build that checks for latin phrases will fail.